pages/common/tailscale.md

```diff
@@ -0,0 +1,36 @@
+# tailscale
+
+> The easiest and most secure way to use WireGuard and 2FA.
+> More information: <https://tailscale.com>.
+
+- Connect to Tailscale:
+
+`sudo tailscale up`
+
+- Disconnect from Tailscale:
+
+`sudo tailscale down`
+
+- Display current Tailscale IP addresses:
+
+`tailscale ip`
+
+- Ping a peer node at the Tailscale layer and display which route it took for each response:
+
+`tailscale ping {{ip|hostname}}`
+
+- Analyze the local network conditions and display the result:
+
+`tailscale netcheck`
+
+- Run a web server for controlling Tailscale:
```

Comment:
Does this start a local server and opens it in the web browser?

Reply:
Just start the server, and displays the URL http://localhost:8088 to enter to it. And the user should use `Ctrl + C` to stop it.

Reply:
```suggestion
- Start a web server for controlling Tailscale:
```